aries-site/src/examples/components/checkbox/CheckBoxValidationExample.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, Form, FormField } from 'grommet';
+
+export const CheckBoxValidationExample = () => {
+  const defaultErrorMessage = 'Check a checkbox/toggle to resolve this error.';
+  const [message, setMessage] = useState(defaultErrorMessage);
+
```

Comment:
I don't think we want each field to share the same state. Recommend having independent states for each.